Stringtable.xml

```diff
@@ -983,7 +983,7 @@
 				<English>Function Auto-completion</English>
 				<Czech>Function Auto-completion</Czech>
 				<French>Function Auto-completion</French>
-				<German>Auto-Vervollständigungs Funktion</German>
+				<German>Auto-Vervollständigungs-Funktion</German>

```

Comment:
Both translations seem to be: `auto-completion function`
but:
Function auto-completion: auto-completion for functions
(if it's correct, plz ignore)